.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Unit tests CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'yarn'
```

Comment:
Should re-trigger the job to learn if the current cache is working correctly nha:
![image](https://user-images.githubusercontent.com/22078891/218691197-066d78bc-a63d-4fad-8064-84de69bcdcab.png)

Reply:
The cache is available in 2nd job
<img width="1102" alt="image" src="https://user-images.githubusercontent.com/44129390/218728822-6eed3ea7-e459-4745-a90f-ed47060877dd.png">